### PR TITLE
added 3 web 

### PR DESCRIPTION
**Pull request by bharatkumarthapa**

### CODE REVIEW

**Review by thenishantsapkota — APPROVED** [review]

**Comment by thenishantsapkota** [comment]
@bharatkumarthapa Please resolve the merge conflicts and we'll be ready to merge this!

File: data/websites.json [hunk 1]
```diff
@@ -244,6 +244,31 @@
...
       "description": "Official Website Of Supreme Court of Nepal"
+    },
+    {
+      "name": "Department of Immigration",
+      "url": "https://www.immigration.gov.np/",
+      "description": "Official Website of Department of Immigration Nepal"
+    },
+    {
+      "name": "Nepal Tourism Board",
+      "url": "hhttps://ntb.gov.np/",
```
Comment (Saphall) [inline data/websites.json:255]:
```suggestion
      "url": "https://ntb.gov.np/",
```

File: data/websites.json [hunk 2]
```diff
@@ -244,6 +244,31 @@
       "name": "Supreme Court",
       "url": "https://supremecourt.gov.np",
       "description": "Official Website Of Supreme Court of Nepal"
+    },
+    {
```
Comment (Saphall) [inline data/websites.json:248]: Please arrange the list alphabetically.

**Comment by TheLearneer** [comment]
Closing due to inactivity, please open another PR if you feel if it's wrong 🙏

**Pull request closed without merging**